docs/api.rst

```diff
@@ -2850,15 +2865,168 @@ The details of each asset can contain the following keys:
                   "cryptocompare":"VET",
                   "coingecko":"vet",
                   "protocol":"None"
+              }
+          },
+          "message": ""
+      }
+
+   :resjson object result: A mapping of asset identifiers to their respective asset details.
+   :statuscode 200: Assets successfully queried.
+   :statuscode 500: Internal rotki error
+
+
+Get asset identifiers mappings
+================================
+
+.. http:post:: /api/(version)/assets/mappings
+
+   Doing a POST on the assets mappings endpoint with a list of of identifiers will return a mapping of those identifiers to their respective name and symbols.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      POST /api/1/assets/mappings HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {
+        "identifiers": [
+            "eip155:1/erc20:0xB6eD7644C69416d67B522e20bC294A9a9B405B31",
+            "DCR",
+            "eip155:1/erc20:0xcC4eF9EEAF656aC1a2Ab886743E98e97E090ed38"
+        ]
+      }
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+          "result": {
+              "eip155:1/erc20:0xB6eD7644C69416d67B522e20bC294A9a9B405B31": {
+                  "name": "0xBitcoin",
+                  "symbol": "0xBTC"
+              },
+              "DCR": {
+                  "name": "Decred",
+                  "symbol": "DCR"
               },
+              "eip155:1/erc20:0xcC4eF9EEAF656aC1a2Ab886743E98e97E090ed38": {
+                  "name": "DigitalDevelopersFund",
+                  "symbol": "DDF"
+              }
           },
           "message": ""
       }
 
+   :resjson object result: A mapping of identifiers to names and symbols.
+   :statuscode 200: Assets successfully queried.
+   :statuscode 400: One of the identifiers are not valid. Provided JSON is in some way malformed.
+   :statuscode 500: Internal rotki error.
+
+Search for assets
+===================
+
+.. http:post:: /api/(version)/assets/search
+
+   Doing a POST on the assets search endpoint will return a list of objects containing an asset's name, symbol and identifier based on the search keyword provided.
```

Comment:
here mention the order by which results are returned. Is it symbol? Is it name? Shouldn't this be configurable? Is is the `search_column` argument?